### DISCUSSION TITLE
Slider/Range Comonent

### DISCUSSION THREAD

**Discussion by Nerixyz** [tailwindlabs/headlessui#579] [Ideas]
It would be nice to have a slider component that can be fully styled and is accessible, because the browsers each have their own style for the range-input.

**Comment by ianstormtaylor** [comment]
Please, please, please. This would be amazing. One of the few that I can't get working with HeadlessUI+TailwindUI+TailwindCSS quickly.

**Comment by aimproxy** [comment]
I need one of these right now!! Some examples on MaterialUI https://mui.com/pt/components/slider/

**Comment by ianstormtaylor** [comment]
I ended up using Radix UI for their sliders, in case it helps anyone else.

**Comment by arlgnv** [comment]
Also looking forward to this component. Meanwhile, ended up using [this](https://mui.com/base-ui/react-slider/)
